README.md

```diff
@@ -35,14 +35,19 @@ cd Actively/views/
 npm start
 ```
 
-### Running Linter
-From root folder
-...
-To run on all files
-...
-./node_modules/.bin/eslint .
-...
-To run on specific file
-...
-./node_modules/.bin/eslint File.js
-...
+
+### Setup the Linter
+```
+npm install eslint --save-dev
+./node_modules/.bin/eslint --init
+(Select Popular Style -> AirBnB -> JSON)
+```
```

Comment:
The linter is set up. The `Setup the Linter` documentation block is unnecessary. Setting up the linter with a different code style will just cost time in long run. Delete this documentation block.